internal/services/redis/redis_cache_resource_test.go

```diff
@@ -534,7 +534,11 @@ func (t RedisCacheResource) Exists(ctx context.Context, clients *clients.Client,
 func (RedisCacheResource) basic(data acceptance.TestData, requireSSL bool) string {
 	return fmt.Sprintf(`
 provider "azurerm" {
-  features {}
+  features {
+    resource_group {
+      prevent_deletion_if_contains_resources = false
```

Comment:
this would indicate something isn't being deleted correctly which would cause issues for real users - rather than adding this workaround, can we fix the issue here?